docs/docs-content/release-notes.md

```diff
@@ -11,6 +11,18 @@ sidebar_custom_props:
 tags: ["release-notes"]
 ---
 
+## November 7, 2023 - Release 4.1.9
+
+This release contains the following minor bug fixes related to protected Helm Chart and Open Container Initiative (OCI) registries.
+
+- Where Palette displayed the UID of private or protected Helm Chart registries in a drop-down menu during cluster profile creation, Palette now correctly displays the registry name.
+
+- The issue is fixed where Palette did not allow charts from private or protected Helm Chart and OCI registries to be attached to cluster profiles. 
+
+- The issue is resolved where Palette did not display packs for selection that belonged to a protected Helm Chart registry, preventing cluster profile creation. 
```

Comment:
@ritawatson I'm a bit confused about this sentence. Is this sentence trying to convey that some packs were not available for selection? And as a result, the user could not create a cluster profile with those packs?

Reply:
Yes.